### DISCUSSION TITLE
Q: 使用多组表情包时底部的切换栏显示不全，请问这个怎么解决？ || Q: When using multiple emoticons, the switching bar at the bottom is not fully displayed. How to solve this problem?

### DISCUSSION THREAD

**Discussion by van23qf** [walinejs/waline#1985] [Q&A] [answered]
### Describe the bug | 问题描述

如图所示

![20230525162632](https://github.com/walinejs/waline/assets/5917036/b2595295-7989-478d-9772-8534f18fa027)

### Website URL | 问题网站

https://www.qfcoder.com/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

as the picture shows

![20230525162632](https://github.com/walinejs/waline/assets/5917036/b2595295-7989-478d-9772-8534f18fa027)

### Website URL | Problem Website

https://www.qfcoder.com/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by Mister-Hope** [comment, marked as answer]
![image](https://github.com/walinejs/waline/assets/33315834/924b9f38-b923-46ce-bff7-31c76ea7230c)


---
[Image]!
